control/src/main/java/org/openbase/bco/dal/control/layer/unit/AbstractUnitController.java

```diff
@@ -159,6 +168,34 @@
     private final Map<ServiceType, Timeout> requestedStateCacheTimeouts;
     final BuilderSyncSetup<DB> builderSetup;
 
+    private static final String INFLUXDB_BUCKET_DEFAULT = "bco-persistence";
+    private static final Integer READ_TIMEOUT = 60;
+    private static final Integer WRITE_TIMEOUT = 60;
+    private static final Integer CONNECT_TIMOUT = 40;
+    private static final String INFLUXDB_URL;
+    private static String INFLUXDB_ORG_ID;
+    private static final String INFLUXDB_PROPERTIES = "influxdb.properties";
+    private static Properties influxDBProperties = new Properties();
+    private static final char[] TOKEN;
+
+    static {
+        loadInfluxDBProperties();
+        TOKEN = influxDBProperties.get("token").toString().toCharArray();
+        INFLUXDB_ORG_ID = influxDBProperties.get("influxdb_org_id").toString();
+        INFLUXDB_URL = influxDBProperties.get("influxdb_url").toString();
+    }
+
+    private static void loadInfluxDBProperties() {
```

Comment:
Nice idea but since we are at the server side you should use the registry meta config entries of the the influx db app unit to load the properties. On server side the registry should be the only place where configurations are stored. This keeps the bco setup simple and allows permission restrictions if required.